openxr/test/cmake/scripts.cmake

```diff
@@ -0,0 +1,161 @@
+# write out a script to invoke monado
+string(REGEX MATCH "tcsh$" TCSHELL $ENV{SHELL})
+string(REGEX MATCH "csh$" CSHELL $ENV{SHELL})
+if(CSHELL)
+    set(SET_CMD "setenv")
+    set(SPACER " ")
+else()
+    set(SET_CMD "export")
+    set(SPACER "=")
+endif()
+
+if(CSHELL)
+    set(MONADO_OUT "${CMAKE_CURRENT_SOURCE_DIR}/illixr_monado_oxr.csh")
+    set(XR_OUT "${CMAKE_CURRENT_SOURCE_DIR}/openxr_ht_test.csh")
+    set(IXR_OUT "${CMAKE_CURRENT_SOURCE_DIR}/illixr_ht_test.csh")
+else()
+    set(MONADO_OUT "${CMAKE_CURRENT_SOURCE_DIR}/illixr_monado_oxr.sh")
+    set(XR_OUT "${CMAKE_CURRENT_SOURCE_DIR}/openxr_ht_test.sh")
+    set(IXR_OUT "${CMAKE_CURRENT_SOURCE_DIR}/illixr_ht_test.sh")
+endif()
+
+macro(write_three TXT)
+    file(APPEND ${MONADO_OUT} "${TXT}")
+    file(APPEND ${XR_OUT} "${TXT}")
+    file(APPEND ${IXR_OUT} "${TXT}")
+endmacro()
+
+file(WRITE ${MONADO_OUT} "")
+file(WRITE ${XR_OUT} "")
+file(WRITE ${IXR_OUT} "")
+if(CSHELL)
+    if(TCSHELL)
+        write_three("#!/usr/bin/env tcsh\n\n")
+    else()
+        write_three("#!/usr/bin/env csh\n\n")
+    endif()
+else()
+    write_three("#!/usr/bin/env sh\n\n")
+endif()
+
+write_three("# set environment variables\n")
+
+write_three("${SET_CMD} IXIR_PTH${SPACER}${CMAKE_INSTALL_PREFIX}/lib\n\n")
```

Comment:
@Jebbly suggested adding another line to print out more error messages, it was helpful for me to debug some stuff:
```
export VK_INSTANCE_LAYERS=VK_LAYER_KHRONOS_validation
```

Reply:
done